src/main/java/eu/bbmri/eric/csit/service/negotiator/mappers/RequestModelsMapper.java

```diff
@@ -26,22 +23,17 @@ public class RequestModelsMapper {
 
   @Autowired ModelMapper modelMapper;
 
-  @Value("${negotiator.frontend-url}")
-  private String FRONTEND_URL;
+  private final String FRONTEND_URL;
```

Comment:
The name should be camelCase https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names